static/app/views/organizationIntegrations/integrationRow.spec.tsx

```diff
@@ -0,0 +1,170 @@
+      render(
+        <IntegrationRow
+          organization={org}
+          type="firstParty"
+          slug="bitbucket"
+          displayName="Bitbucket"
+          status="Installed"
+          publishStatus="published"
+          configurations={1}
+          categories={[]}
+        />
+      );
+      expect(screen.getByText('Bitbucket')).toBeInTheDocument();
+      expect(screen.getByText('Bitbucket')).toHaveAttribute(
+        'href',
+        `/settings/${org.slug}/integrations/bitbucket/`
+      );
+      expect(screen.getByText('1 Configuration')).toBeInTheDocument();
+    });
+
+    it('has been installed (3 configurations)', function () {
+      render(
+        <IntegrationRow
+          organization={org}
+          type="firstParty"
+          slug="bitbucket"
+          displayName="Bitbucket"
+          status="Installed"
+          publishStatus="published"
+          configurations={3}
+          categories={[]}
+        />
+      );
+      expect(screen.getByText('Installed')).toBeInTheDocument();
+      expect(screen.getByText('Bitbucket')).toHaveAttribute(
+        'href',
+        `/settings/${org.slug}/integrations/bitbucket/`
+      );
+      expect(screen.getByText('3 Configurations')).toBeInTheDocument();
+    });
+
+    it('has not been installed', function () {
+      render(
+        <IntegrationRow
+          organization={org}
+          type="firstParty"
+          slug="github"
+          displayName="Github"
+          status="Not Installed"
+          publishStatus="published"
+          configurations={0}
+          categories={[]}
+        />
+      );
+      expect(screen.getByText('Not Installed')).toBeInTheDocument();
+      expect(screen.getByText('Github')).toHaveAttribute(
+        'href',
+        `/settings/${org.slug}/integrations/github/`
+      );
+    });
+  });
+  describe('Plugin', function () {
+    it('has been installed (1 project)', function () {
+      render(
+        <IntegrationRow
+          organization={org}
+          type="plugin"
+          slug="twilio"
+          displayName="Twilio (SMS) "
+          status="Installed"
+          publishStatus="published"
+          configurations={1}
+          categories={[]}
+        />
+      );
+      expect(screen.getByText('Installed')).toBeInTheDocument();
+      expect(screen.getByText('1 Configuration')).toBeInTheDocument();
+      expect(screen.getByText('Twilio (SMS)')).toHaveAttribute(
+        'href',
+        `/settings/${org.slug}/plugins/twilio/`
+      );
+    });
+
+    it('has been installed (3 projects)', function () {
+      render(
+        <IntegrationRow
+          organization={org}
+          type="plugin"
+          slug="twilio"
+          displayName="Twilio (SMS) "
+          status="Installed"
+          publishStatus="published"
+          configurations={3}
+          categories={[]}
+        />
+      );
+      expect(screen.getByText('Installed')).toBeInTheDocument();
```

Comment:
Maybe would be good to add:
```suggestion
      expect(screen.getByText('3 Configurations')).toBeInTheDocument();
      expect(screen.getByText('Installed')).toBeInTheDocument();
```

to match the now-removed tests